frontend/src/pages/Publish.tsx

```diff
@@ -86,7 +93,9 @@ const Publish = () => {
           placeholder="Tell your story..."
           className="tracking-wide text-[#0B1215] font-light"
           value={content}
-          onChange={setContent}
+          onChange={(value) =>
+            setContent(htmlTagRegex.test(value) ? "" : value)
```

Comment:
A simple `value.trim().length > 0` could've worked here. (Just like title)
Any specific reason why an HtmlTag validation for content here ?

Reply:
actually, there are additional tags getting generated in content. When we first start typing the content, `<p>` tags are generated and when you remove them, the value is content is still `<p><br></p>`
So, to handle this scenario I added a regex for html tags.[
![Screenshot 2024-05-26 at 1 38 38 PM](https://github.com/aadeshkulkarni/medium-app/assets/47346214/b75519e0-0668-40e4-9333-d05af0d7b43e)
![Screenshot 2024-05-26 at 1 38 29 PM](https://github.com/aadeshkulkarni/medium-app/assets/47346214/256f7d40-4ec6-4140-9464-ebbb273419da)

Reply:
Alright. Makes sense. Wasn't aware of this. Thanks for sharing.